src/main/kotlin/net/ccbluex/liquidbounce/utils/aiming/anglesmooth/NoneAngleSmoothMode.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2024 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ *
+ *
+ */
+package net.ccbluex.liquidbounce.utils.aiming.anglesmooth
+
+import net.ccbluex.liquidbounce.config.ChoiceConfigurable
+import net.ccbluex.liquidbounce.utils.aiming.Rotation
+import net.ccbluex.liquidbounce.utils.aiming.RotationManager
+import net.minecraft.entity.Entity
+import net.minecraft.util.math.Vec3d
+import kotlin.math.roundToInt
+
+class NoneAngleSmoothMode(override val parent: ChoiceConfigurable<*>) : AngleSmoothMode("None") {
```

Comment:
I think None in terms of "Angle Smooth Mode" would stand for "No smoothing". But that mode practically disables the whole aiming. Instead you can make a boolean option, that says it will fully disable the rotation manager (for this module) and simply return on aimAt.

Reply:
alright, will work on it tommorow